test/functional/bootstrap.js

```diff
@@ -182,6 +182,10 @@ before(async () => {
     // using --no-sandbox.
     const args = [];
 
+    if (process.env.HTTPS_PROXY) {
+        args.push(`--proxy-server=${process.env.HTTPS_PROXY}`);
+    }
```

Comment:
This breaks everything for me now. What I propose is this instead:

```js
const isProxy = process.argv.pop().match('--proxy');
if (proxy && process.env.HTTPS_PROXY) {
    args.push(`--proxy-server=${process.env.HTTPS_PROXY}`);
}
```

And now you can use `npm run test-functional -- --proxy` if you need a proxy.

Reply:
nevermind, I got a bad `HTTPS_PROXY` variable.